<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ƒêƒÉng nh·∫≠p</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
     QWidget{
        background:#0f172a;
        color:white;
        font:12pt &quot;Segoe UI&quot;;
     }
     QLineEdit{
        background:#020617;
        border:1px solid #334155;
        border-radius:6px;
        padding:8px;
        color:white;
     }
     QPushButton{
        border-radius:6px;
        padding:10px;
     }
     #btnLogin{
        background:#2563eb;
     }
     #btnGoogle{
        background:#111827;
        border:1px solid #374151;
     }
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="credit_btn">
      <property name="styleSheet">
       <string notr="true">#credit_btn{
background:#2563eb;
border-radius:6px;
padding:10px;
}
#credit_btn:hover{
background:rgb(29, 80, 189);
}</string>
      </property>
      <property name="text">
       <string>Credit</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>120</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string>font-size:40px;</string>
        </property>
        <property name="text">
         <string>
üß†
          </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string>font-size:22px;font-weight:600;</string>
        </property>
        <property name="text">
         <string>ƒêƒÉng nh·∫≠p</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string>color:#9ca3af;</string>
        </property>
        <property name="text">
         <string>Ch√†o m·ª´ng quay tr·ªü l·∫°i l·ªõp h·ªçc</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>420</width>
          <height>280</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QFrame{
           background:#020617;
           border:1px solid #1f2937;
           border-radius:12px;
          }
QPushButton{
background-color:#0F172A;
border-radius:4px;
border:1px solid #374151
}
QPushButton:hover{
background-color:rgb(38, 59, 107)
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>T√™n gi√°o vi√™n</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name2">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>V√≠ d·ª•: L√≤ VƒÉn T√πng</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>M·∫≠t kh·∫©u</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pass2">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="signin">
           <property name="text">
            <string>ƒêƒÉng nh·∫≠p</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="didnt_have">
           <property name="text">
            <string>Ch∆∞a c√≥ t√†i kho·∫£n? ƒêƒÉng k√Ω ngay </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
